consoleApp/src/main/java/com/issoft/store/StoreApp.java

```diff
@@ -42,13 +42,43 @@ else if(userCommand.equalsIgnoreCase("top")){
         else if(userCommand.equalsIgnoreCase("exit")) {
                 reader.close();
                 System.out.println("See you soon!");
-                i = 0;
+                System.exit(0);
             }
```

Comment:
I was expecting to see new command"create order" where user can select which product will be added to purchaseGoods collection